src/qt/openuridialog.h

```diff
@@ -7,6 +7,8 @@
 
 #include <QDialog>
 
+#include <qt/platformstyle.h>
```

Comment:
https://github.com/bitcoin/bitcoin/pull/17955/files#r368907744:
> Just forward declare `PlatformStyle`.

Reply:
You mean forward declare it here, and move this include to openuridialog.cpp? Otherwise there is "invalid use of incomplete type" compiler error.

Reply:
> You mean forward declare it here, and move this include to openuridialog.cpp?

Yeap.